python/lsst/eotest/sensor/BFTask.py

```diff
@@ -158,16 +156,22 @@ def write_eotest_output(self, BFResults, sensor_id, meanidx=0, adu_max=1e5):
         units = []
         columns = []
         for amp in BFResults:
-            colnames.extend(['AMP%02i_XCORR' % amp, 'AMP%02i_XCORR_ERR' % amp,
-                             'AMP%02i_YCORR' % amp, 'AMP%02i_YCORR_ERR' % amp,
+            colnames.extend(['AMP%02i_COV10' % amp, 'AMP%02i_COV10_ERR' % amp,
```

Comment:
Could you update the [plotting code](https://github.com/lsst-camera-dh/eotest/blob/BFTask_Updates/python/lsst/eotest/sensor/EOTestPlots.py#L941) in `EOTestPlots.bf_curves` to account for these new column names?